multisig/proposals/vip-008/vip-008-sepolia.ts

```diff
@@ -0,0 +1,53 @@
+import { makeProposal } from "../../../src/utils";
+
+const PRIME_LIQUIDITY_PROVIDER = "0x4fCbfE445396f31005b3Fd2F6DE2A986d6E2dCB5";
+const VTreasury = "0x4116CA92960dF77756aAAc3aFd91361dB657fbF8";
+const PRIME = "0x27A8ca2aFa10B9Bc1E57FC4Ca610d9020Aab3739";
+
+const ETH = "0x700868CAbb60e90d77B6588ce072d9859ec8E281";
+const BTC = "0x92A2928f5634BEa89A195e7BeCF0f0FEEDAB885b";
+const USDC = "0x772d68929655ce7234C8C94256526ddA66Ef641E";
+const USDT = "0x8d412FD0bc5d826615065B931171Eed10F5AF266";
+
+export const vip008 = () => {
+  return makeProposal([
+    {
+      target: PRIME_LIQUIDITY_PROVIDER,
+      signature: "setTokensDistributionSpeed(address[],uint256[])",
+      params: [
+        [ETH, BTC, USDC, USDT],
+        ["24438657407407", "1261574074074", "35491568287037037", "87191261574074074"],
```

Comment:
The decimals of the tokens in sepolia are:
* BTC: 8
* ETH: 18
* USDC: 6
* USDT: 6

I think we are assuming now they all have 18 decimals